lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -39,7 +39,7 @@
 
 # HTML for registration button
 REG_STR = "<form id=\"class_enroll_form\" method=\"post\" data-remote=\"true\" action=\"/change_enrollment\">"
-SHIB_ERROR_STR = "The currently logged-in user account does not have permission to enroll in this course."
+SHIB_ERROR_STR = "The currently logged\\u002Din user account does not have permission to enroll in this course."
```

Comment:
Nit: The following is probably more clear without requiring explanation:
```
SHIB_ERROR_STR = js_escaped_string("The currently logged-in user account does not have permission to enroll in this course.")
```
Also, was this error tested manually?  Just curious.